css homework

Kopeyka885 opened the pull request:
A journey of a thousand miles starts with a single step (C) Lao Tzu

Kopeyka885 committed:
css homework

A journey of a thousand miles starts with a single step (C) Lao Tzu

fromanywhere commented on css/todo-app.css:
```diff
@@ -0,0 +1,35 @@
+.todo-app {
+	width: 56.2%;
+	margin: 0 auto;
```
подумайте, пожалуйста, о минимальной ширине Вашего приложения

fromanywhere commented on css/reset.css:
```diff
@@ -0,0 +1,14 @@
+
+/* Remove outline on the inputs and buttons (Мешает тестировать кнопки и инпуты TAB-ом(но всё работает,
+отключал это свойство во время процесса))*/
+:active, :hover, :focus {
```
Оставьте его, без него не понятно, где я сейчас нахожусь

fromanywhere commented on css/main-content__tasks.css:
```diff
@@ -0,0 +1,77 @@
+}
+
+.list-item__view:hover > .list-item__delete-task{
+	z-index: 1;
```
на фокус тоже, пожалуйста

fromanywhere replied:
Ну вот не работает же, потому что .list-item__delete-task:focus будет

fromanywhere commented on css/actions.css:
```diff
@@ -0,0 +1,47 @@
+}
+
+.tasks-clear-completed{
+	background-color: #fff;
```
на кнопке кегль и гарнитура отличаются от соседних элементов, проверьте, пожалуйста

fromanywhere commented on css/main-content__add-input.css:
```diff
@@ -0,0 +1,39 @@
+	font-weight: 300;
+	flex-grow: 1;
+	padding: 0 5px;
+	overflow: hidden;
```
но зачем

fromanywhere commented on css/main-content__tasks.css:
```diff
@@ -0,0 +1,77 @@
+}
+
+.list-item__text{
+	padding-left: 15px;
```
прилипает, предусмотрите отступы 
![image](https://user-images.githubusercontent.com/1042969/67621906-16b1c580-f81d-11e9-9cbd-d9505313db20.png)

fromanywhere commented on css/main-content__tasks.css:
```diff
@@ -0,0 +1,77 @@
+	background-color: #FAFAFA;
+}
+
+.list-item__view:hover > .list-item__delete-task{
```
не думаю, что здесь нужен >

fromanywhere reviewed (changes requested):
Вообще довольно прилично, вроде бы кроме этих замечаний я ничего не нашел

Kopeyka885 committed:
Big changes

Установил минимальный размер приложения, добавил фокус, убрал удаление подсветки, оптимизировал установление шрифтов, все кегли и гарнитуры в нижней части совпадают, скорректировал селектор

fromanywhere commented on css/main-content__tasks.css:
```diff
@@ -0,0 +1,76 @@
+	background-image: url(../svg/checked.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+}
```
на фокус сделайте тоже обводку потемнее, пожалуйста

fromanywhere reviewed (changes requested):
Мне не очень нравится поведение футера сейчас, но да ладно, это непринципиально. Поправьте, пожалуйста, замечания по фокусам

Kopeyka885 committed:
Focuses&footer changes

Исправил странное поведение подвала, добавил фокусы (надеюсь, на этот раз я вас правильно понял)
